hack/README.md

```diff
@@ -0,0 +1,70 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used to build and manage this
+repository. If there are any issues regarding the intention of a particular
+script (or even part of a certain script), please reach out to us.
+It may help us either refine our current scripts, or add on new ones
+that are appropriate for a given use case.
+
+## DinD (dind.sh)
+
+DinD is a wrapper script which allows Docker to be run inside a Docker
+container. DinD should only be executed inside a Docker
```

Comment:
"should only" could probably be stated as "requires the container to be run with privileged mode enabled"